src/Widgets/NotificationEntry.vala

```diff
@@ -141,7 +142,15 @@ public class Notifications.NotificationEntry : Gtk.ListBoxRow {
         deck.add (delete_right);
         deck.visible_child = grid;
 
-        add (deck);
+        revealer = new Gtk.Revealer () {
+            reveal_child = true,
+            transition_duration = 195,
```

Comment:
How specific, but not arguing with this :p

Reply:
I actually have no idea where this number came from. It looks like this should probably be 200 based on https://github.com/elementary/granite/issues/429